doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -2956,6 +2956,26 @@ The API can also be used to reset the dataset to use the default/inherited value
 
 The default will always be the same provider as for the dataset PID if that provider can generate new PIDs, and will be the PID Provider set for the collection or the global default otherwise.
 
+Reconcile the PID ofa a dataset (If multiple PIDProvider are Enabled)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Dataverse can be configured to use multiple PID Providers (see the :ref:`pids-configuration` section for more information).
+It is possible to update the used PIDProvider of a dataset. This API not not modifies the PIDProvider, but
```

Comment:
Suggested change: It is possible to update the PIDProvider used by a dataset and assign the dataset a new PID that uses the protocol/authority/separator/shoulder of the new PIDProvider. This API 'reconciles' a draft dataset's PID by creating a new PID supported by the PidProvider and assigning the original PID as an alternativePersistentIdentifier for the dataset. The API is restricted to datasets that have not already been published. (It does not make any changes to any PidProvider.)